frontend/src/Components/Conditions/InfoButton.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import '../../css/InfoButton.css';
+
+const InfoButton: React.FC = () => {
```

Comment:
```suggestion
/**
 * A small button showing information and instructions to use the webapp
 */
const InfoButton: React.FC = () => {
```
Please add description for the documentation